### ISSUE TITLE
[Design] 10-10EZR - Standalone form: Edit Household Info design

### ISSUE THREAD

**Issue by hdjustice** [department-of-veterans-affairs/va.gov-team#63823]
## Background
Now that the decision has been made to build a standalone 10-10EZR form, we need to design page(s) to allow the Veteran to edit their dependent information, inline with the Section III of the paper form - Household Information. 
**Note** - This section would include Spouse and Child dependents, Income, and Expenses.

[Private Zenhub Image](https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBbklTIiwiZXhwIjpudWxsLCJwdXIiOiJibG9iX2lkIn19--bd42c60042e35a37262754d11ce29f2a481a509d/image.png)

## Tasks
- [x] Review Section III of the paper form
- [x] Design pages editing the household information
- [x] Review with the team/PO/PM

## Acceptance Criteria
- Edit Household Info designs are created and shared with the team

**Comment by jessicastump** [comment]
[Sketch page](https://www.sketch.com/s/912cab8e-d234-44dd-be1f-2bedb3f50b22/p/3B1847D9-15A8-4526-A264-F00F656EDAC6/canvas) of household information section. This includes spouse, dependents and income/deductible pages.

**Comment by hdjustice** [comment]
From Jessica:
[Sketch page](https://www.sketch.com/s/912cab8e-d234-44dd-be1f-2bedb3f50b22/p/3B1847D9-15A8-4526-A264-F00F656EDAC6/canvas) of household information section. This includes spouse, dependents and income/deductible pages.

**Comment by jessicastump** [comment]
Adding [a link to the corresponding content ticket](https://github.com/department-of-veterans-affairs/va.gov-team/issues/65056#event-10299648879)

**Comment by jessicastump** [comment]
The [Sketch file](https://www.sketch.com/s/912cab8e-d234-44dd-be1f-2bedb3f50b22/p/3B1847D9-15A8-4526-A264-F00F656EDAC6/canvas) has been updated with all the pages and alerts within the household information section. This aligns with the recent EZ household updated content and is consistent there (questions, additional info context, etc). 
Things to note:
1. For the on page pre-fill design system alert (blue box) we discussed that this should be on the first question/page of each step of the form and not used throughout. For this section, that would be the marital status question. 
2. The information for income and deductibles/expenses **will not** pre-fill with previous year's data. The content is specific for 2022, so unless there is 2022 data available in ES, this should not pre-fill. 
3. Signed up for Platform office hours to discuss alerts in general for Sept 19 to talk about ezr overall, but part of that alert convo is the v3 component for warning alerts on mobile (dependents section, new multi-response pattern). Confirm how this will work in development vs Sketch templates to allow for character limit of buttons within the modal so they are not being cutoff.  
4. [Sketch file](https://www.sketch.com/s/912cab8e-d234-44dd-be1f-2bedb3f50b22/a/nQgj84d) also includes post mvp / V2 draft of a short form possibility for those who have a disability rating over 50% (low priority and will probably be another ticket)
